packages/@uppy/aws-s3-multipart/src/MultipartUploader.js

```diff
@@ -150,6 +150,8 @@ class MultipartUploader {
       return result
     }).then(({ url }) => {
       this._uploadPartBytes(index, url)
+    }).catch((err) => {
```

Comment:
This is an extremely small nit and I almost feel bad for bringing it up: I think this might be a little clearer using the second argument to `.then()`. The `._uploadPartBytes()` call is conceptually the next step after the Promise chain ends, having another `.catch()` _reads_ like it is part of the chain. Basically, I could see myself reading this in the future and thinking the `this._uploadPartBytes()` call is missing a return statement.

I would prefer doing this:
```js
.then(({ url }) => {
  this._uploadPartBytes(index, url)
}, (err) => {
  this._onError(err)
})
```

Then it's clear that this is the end of the chain and that we have two ways of handling the "final" promise state, if that makes sense.

Reply:
Makes sense! I pushed a fix :)